Add feature to return elements array

Khalid7274 opened the pull request:
Hi Sir,

This pull request introduces a new feature that allows users to return an array of elements based on specific criteria. The main changes include:

New Functionality: Implemented a function to filter and return elements from an array.
Second Functionality: to return unique value from and existing arrary.

Thanks, kindly let me know if you have any revision.

Khalid7274 committed:
Assignment of two arrays to find values in existing array and return unique value added

AH82021 commented on js/app.js:
```diff
@@ -164,65 +165,36 @@ function reverseArray(arr){
 
-console.log(filterdata);
 
+//Nkhalid Assignment
```
Please  create a function for each functionaliry  , look  Jeena's code (funSum() & findMax()) and follows the same pattern.Thanks

AH82021 reviewed (changes requested):
Please create a function for each functionaliry , look Jeena's code (funSum() & findMax()) and follows the same pattern.Thanks

Khalid7274 committed:
functions created to check if any values exist and to find duplicate value

Khalid7274 commented:
Thank you, Ustad, for your comment. I have revised the code and pushed the changes to the same branch!

This pull request introduces a new function that removes duplicate values from an array and returns a new array with only unique values. The main changes include:
New Functionalities: Implemented the removeDuplicates function to filter out duplicate values from an array. Additionally, added functionality to find elements in the array.

This version corrects the grammar and improves the clarity of your message. Let me know if you need any more help!
